vars/snyk.groovy

```diff
@@ -27,10 +26,22 @@ def snykTest(Map snykTestArgs=[:]) {
             break;
     }
     withCredentials([string(credentialsId: "snyk-sa-token", variable: "TOKEN")]) {
-    sh """
-        snyk auth ${TOKEN}
-        ${xrayCommand}
-    """
+        sh """
+            snyk auth ${TOKEN}
+            ${xrayCommand}
+        """
+    }
+}
+
+def snykIac(Map snykTestArgs=[:]) {
+    logger.info("Snyk IAC")
+    logger.info("Technology: ${snykTestArgs.tech}")
+    logger.info("Testing terraform files...")
+    withCredentials([string(credentialsId: "snyk-sa-token", variable: "TOKEN")]) {
+        sh """
+            snyk auth ${TOKEN}
+            snyk iac test ${snykTestArgs.deploy.workDir} --org=${snykTestArgs.snyk.orgId} --report || true
```

Comment:
Wouldn't this always pass?

Reply:
Yes, it always passes because if there's any findings at all the pipeline will fail. We still have the manual approval step in the pipeline, so nothing will get deployed without someone checking the report first.

Reply:
I like having the pipeline fail when the test fails, but lets see what feedback we get. Maybe just with a "--severity-threshold=high" ?